### DISCUSSION TITLE
Playlist with start and end times.

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#4251] [General]
In centova cast there is a setting in which I mark "play until the end" and in this option I don't need to put the start and end time of a playlist. Already in azuracast I didn't find a way that I just put the start time and it plays until it finishes the playlist and proceeds with the programming. Is there a way to put only the start time of a playlist and not put the end time?

**Comment by silvaalex2021** [comment]
I would love it, too!

I've suggested this a few times, but I haven't had any feedback so far.
